app/src/main/java/com/example/byitscover/scrapers/BarnesAndNobleScraper.java

```diff
@@ -43,50 +42,25 @@ public BarnesAndNobleScraper() {
      * @throws IOException
      */
     public List<BookListing> scrape(Query query) throws IOException {
-        List<Result> results = ScraperHelper.googleAPISearch(ScraperConstants.BARNES_AND_NOBLE, query);
-        String searchingUrl = getActualBookResult(results);
-        Document document = Jsoup.connect(searchingUrl).get();
-        System.out.println(searchingUrl);
+        String url = getUrlWithQuery(query);
+        System.out.println(url);
```

Comment:
Should this print be removed?

Reply:
I find it helpful as a logging sort of thing since we don't have it set up for actual logging